native/src/hooks/useEmbeddedOffers.ts

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from 'react'
+
+import {
+  CategoryModel,
+  LoadSprungbrettJobReturn,
+  SPRUNGBRETT_OFFER,
+  loadSprungbrettJobs,
+  useLoadAsync,
+} from 'api-client'
+
+import { determineApiUrl } from '../utils/helpers'
+
+type EmbeddedOffers = {
+  sprungbrett?: Awaited<LoadSprungbrettJobReturn>
```

Comment:
Optional: I think this type would be easier to read if you would change `LoadSprungbrettJobReturn` to just the object without the promise and do this here:
```suggestion
  sprungbrett?: LoadSprungbrettJobReturn
```